src/APM/dev/SIM100.cpp

```diff
@@ -35,17 +35,17 @@ int SIM100::sendMessage(uint8_t dataLength, uint8_t *payload, IO::CANMessage &re
        This entire function will be updated for better handling with CAN Open integration so this
        is a temporary fix */
 
-    if (!expectResponse) {
-        return 0;
-    }
-
-    auto response = can.receive(&responseMessage, true);
-    while (response == nullptr || responseMessage.getId() != CAN_RESPONSE_ID  ||
-           responseMessage.getPayload()[0] != requestMuxByte) {
-        response = can.receive(&responseMessage, false);
-        EVT::core::time::wait(500);
-        // TODO: Implement timeout and return something accordingly.  Also better error handling
-    }
+//    if (!expectResponse) {
```

Comment:
Should this commented code be left in? If so add a comment explaining why the code is commented out.